src/index.js

```diff
@@ -1,17 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
+import App from './App'
+import Parse from "parse"
+
+Parse.initialize(process.env.REACT_APP_API_KEY, process.env.REACT_APP_JS_KEY);
+Parse.serverURL = "https://parseapi.back4app.com/";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
   document.getElementById('root')
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+  )
+  
+  
+  
+  
+  
+//  API KEY: "y1pkFZswJRDe8jhue98PGNh4mxxu5geUNCvNKXK1"
+//   JS_KEY: "HrNmgVEuGQ8macNhM9BbXfkORi4wJOr5HFFks8O6"
```

Comment:
Leaving your keys in comments defeats the purpose of putting them in the .env-files. ;)

Reply:
just leftovers from creating the env files on each individual machine ;)